# In this scenario:
# - the conveyor belt is not used
# - there is one order consisting of two types of parts, found in the bins
# - the first part will fall to the ground once over the kit tray

options:
  insert_models_over_bins: true
  fill_demo_tray: false
  gazebo_state_logging: true
  model_type_aliases:
    order_part1: piston_rod_part
    order_part2: gear_part

orders:
  order_0:
    announcement_condition: time
    announcement_condition_value: 0.0
    parts:
      part_0:
        type: order_part1
        pose:
          xyz: [0.1, -0.2, 0]
          rpy: [0, 0, 0]
      part_1:
        type: order_part1
        pose:
          xyz: [-0.1, -0.2, 0]
          rpy: [0, 0, 0]
      part_2:
        type: order_part2
        pose:
          xyz: [0.15, 0.15, 0]
          rpy: [0, 0, 0]
      part_3:
        type: order_part2
        pose:
          xyz: [-0.15, 0.15, 0]
          rpy: [0, 0, 0]
      part_4:
        type: order_part2
        pose:
          xyz: [0.0, 0.15, 0]
          rpy: [0, 0, 0]

drops:
  drop_regions:
    agv1:
      min:
        xyz: [0.0, 2.7, 0.6]
      max:
        xyz: [0.6, 3.5, 2.0]
      destination:
        xyz: [0.6, 3.1, 1.0]
      part_type_to_drop: order_part1
    agv2:
      min:
        xyz: [0.0, -2.7, 0.6]
      max:
        xyz: [0.6, -3.5, 2.0]
      destination:
        xyz: [0.6, -3.1, 1.0]
      part_type_to_drop: order_part1

models_over_bins:
  bin7:
    models:
      piston_rod_part:
        xyz_start: [0.1, 0.1, 0.0]
        xyz_end: [0.5, 0.5, 0.0]
        rpy: [0, 0, 'pi/4']
        num_models_x: 3
        num_models_y: 4

  bin8:
    models:
       gear_part:
        xyz_start: [0.1, 0.1, 0.0]
        xyz_end: [0.5, 0.5, 0.0]
        rpy: [0, 0, 0]
        num_models_x: 4
        num_models_y: 4

time_limit: -1
